prelude.yaml

```diff
@@ -34,6 +34,8 @@ common:
         text: 'Удалите. Уже включено в {0}.'
       - lang: sv
         text: 'Ta bort. Redan inkluderat i {0}.'
+      - lang: tr_TR
+        text: 'Sil. Zaten {0} içinde yer alıyor.'        
```

Comment:
There are empty spaces at the end / after `text: '...'` that need to be removed.
![grafik](https://github.com/loot/prelude/assets/10406657/aa458fa1-787b-4242-9e7a-667f188c0fd8)

Please also check all the other strings too, since there are multiple translations, where these empty spaces at the end need to be removed.